Remove docker-helmfile reference from README (#57)

AdithyaSrivastava01 opened the pull request:
Removed the Reference to resolve README(#57)

dduportal reviewed (approved):
What is the reason for removing the link?

AdithyaSrivastava01 commented:
As https://github.com/jenkins-infra/docker-helmfile repository has been archived this January, the README shouldn't reference it anymore.

AdithyaSrivastava01 commented:
> > As https://github.com/jenkins-infra/docker-helmfile repository has been archived, it shouldn't be referenced in the README file
> 
> In that case, may I suggest to fix the whole sentence? Only removing the link but keeping the rest is the same as keeping a link to an archived repository.

I have added the Dockerhub link in the README.md and replaced it with the previous GitHub link present. Hope it helps.

AdithyaSrivastava01 commented:
Added the latest Docker Image in production and referenced the GitHub repo that contains the build config

dduportal reviewed (approved):
Nice job, thanks!

AdithyaSrivastava01 commented:
Thank You! Hope issue #57 is resolved
